keygen/src/keygen.rs

```diff
@@ -390,6 +392,39 @@ fn app<'a>(num_threads: &'a str, crate_version: &'a str) -> Command<'a> {
                         .help("Overwrite the output file if it exists"),
                 ),
         )
+        .subcommand(
+            Command::new("bls_pubkey")
```

Comment:
can someone count the rest of the snake-cased subcommands in the monorepo please?

Reply:
How do you want it renamed?